src/internal/context/split-panel-context.ts

```diff
@@ -45,17 +43,15 @@ export const SplitPanelContext = createContext<SplitPanelContextProps>({
   size: 0,
   getMaxWidth: () => 0,
   getMaxHeight: () => 0,
-  getHeader: () => null,
   isOpen: true,
   isMobile: false,
   isForcedPosition: false,
   lastInteraction: undefined,
-  splitPanelRef: undefined,
-  splitPanelHeaderRef: undefined,
   onResize: () => {},
   onToggle: () => {},
   onPreferencesChange: () => {},
   reportSize: () => {},
+  reportHeaderHeight: () => {},
```

Comment:
Duplication was here. `reportSize` provided into about split panel size, but also refs to do the same.

Let's use only the `reportSize` pattern, plus add `reportHeaderHeight` to do the same for header